app/models/spree/inventory_unit_decorator.rb

```diff
@@ -0,0 +1,11 @@
+module Spree
+  InventoryUnit.class_eval do
+    def exists_active_return_authorization?
```

Comment:
Looks like this could be Rubyfied a bit – `return_items.map(&:return_authorization).map(&:canceled).include?(false)`